packages/app-bridge/src/react/useBulkDownload.spec.tsx

```diff
@@ -0,0 +1,123 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
+import { act, cleanup, renderHook, waitFor } from '@testing-library/react';
+import sinon from 'sinon';
+import { getAppBridgeBlockStub } from '../tests';
+import { BulkDownloadState, useBulkDownload } from './';
+
+const appBridgeError = new Error('Something went wrong');
+let appBridgeStub = getAppBridgeBlockStub();
+
+describe('useBulkDownload', () => {
+    beforeEach(() => {
+        appBridgeStub = getAppBridgeBlockStub();
+    });
```

Comment:
Please use the same approach as in [this file](https://github.com/Frontify/brand-sdk/blob/main/packages/app-bridge/src/react/usePageTemplateSettings.spec.ts) to generate the hook.
having a `let` in the wild isn't that great